pkg/maps/lbmap/lbmap.go

```diff
@@ -398,6 +399,12 @@ func (*LBBPFMap) UpdateSourceRanges(revNATID uint16, prevSourceRanges []*cidr.CI
 
 	srcRangeMap := map[string]*cidr.CIDR{}
 	for _, cidr := range sourceRanges {
+		if ip.IsIPv6(cidr.IP) && !ipv6 {
+			log.WithFields(logrus.Fields{
+				logfields.ServiceID: revNATID,
+			}).Error("Source range's IP family does not match with the LB's")
+			continue
```

Comment:
Can you add a comment explaining why this code needs to exist? Mentioning that the K8s doesn't catch this should be good enough.

Reply:
Can you also add the justification in the PR description to the commit msg as well?

Reply:
Updated.